java_socketio_chatroom_server/src/test/java/io/github/marcodiri/java_socketio_chatroom_server/ServerWrapperTest.java

```diff
@@ -14,6 +21,15 @@ public void setup() {
         serverWrapper = new ServerWrapper();
     }
 
+    @After
+    public void alwaysStopServer() {
+        try {
+            serverWrapper.stopServer();
+        } catch (Exception ignored) {
+
```

Comment:
maybe call `fail()` on exception?

Reply:
yes, good idea. if it enters the catch block the tests will fail and the exception will still be ignored because it is not thrown by our methods